app/views/results/_filters.html.erb

```diff
@@ -21,6 +21,7 @@
             <%= label_tag "filter[gender]", t('results.show_only_results_by_females') %>
           </p>
 
+          <%= hidden_field_tag "locale", I18n.locale %>
```

Comment:
Should this not be sourcable from the controller? Might be a better way of doing it

Reply:
Ah sure, maybe I should create a @locale variable in the controller and set it equal to this?